tests/test_shell_tools.py

```diff
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: MIT
```

Comment:
possibly, this file should be renamed to `test_cli.py`. (if you like the current name better, leave it as-is.)